lib/cmxl/fields/floor_limit_indicator.rb

```diff
@@ -0,0 +1,24 @@
+module Cmxl
+  module Fields
+    class FloorLimitIndicator < Field
+      self.tag = 34
+      self.parser = /(?<currency>[a-zA-Z]{3})(?<debit_mark>[DC]?)(?<amount>[\d|,|\.]{4,15})/i
+
+      def credit?
+        data['debit_mark'].empty? || data['debit_mark'] == 'C'
```

Comment:
should we do something like this here, too? 
````
data['debit_mark'].to_s.casecmp('C').zero?
```

Reply:
I thought about something like this too, but the condition is, it's NOT credit if it has a 'D', which means 
`data['debit_mark'].to_s.casecmp('D').zero?`
would be the check. 

for me, it would seem odd to have a D in the `credit?` check when reading the code for the first time. so either add a comment to clarify or leave it like this, self-explaining

Reply:
same goes for `debit?`. it's true unless the debit_mark is a `C`.

the identifier must be changed to `type_mark` though

Reply:
yep, true. should be type_mark. :+1:

Reply:
i did change it to `type_indicator`

Reply:
yep, saw it... even better!